quinn-h3/src/proto/connection.rs

```diff
@@ -291,6 +324,7 @@ mod tests {
     impl Default for Connection {
         fn default() -> Self {
```

Comment:
Could this dispatch to `Connection::new` to avoid duplication and risk of divergence?

Reply:
Yeah, I hesitated to get rid of that. Will do.

Reply:
Rebased into "H3: store side information in proto::connection "